requirements.txt

```diff
@@ -1,6 +1,7 @@
-transformers==4.21.1
-fastapi==0.79.0
-uvicorn==0.18.2
+transformers==4.23.1
+fastapi==0.85.1
+uvicorn==0.19.0
 nltk==3.7
-torch==1.12.1
+torch==1.13.0
+--extra-index-url https://download.pytorch.org/whl/cu117
```

Comment:
I see that this `--extra-index-url` is not needed, according to PyTorch website, we can leave it just in case:
- CUDA 11.6
![image](https://user-images.githubusercontent.com/17197768/198968584-52823c14-4cbe-4abd-a84d-a577ad70efa3.png)
- CUDA 11.7
![image](https://user-images.githubusercontent.com/17197768/198968723-9e838c0c-9c18-4db7-b5dd-2f2fdc8737cc.png)